Delegate size (#6879)
harness/developer-hub: docs/platform/delegates/install-delegates/overview.md

   ```

7. Run the command.

> 

Reviewer: This content might work better in the Delegate overview topic. Delegate sizes are documented here: https://developer.harness.io/docs/platform/delegates/delegate-concepts/delegate-overview/#delegate-sizes

:::info
Requirements:

> - **Baseline Configuration:** For clusters with up to 200 nodes and 4000 pods, each delegate should be configured with 2 vCPUs and 8 GB of memory

Reviewer: ```suggestion
- **Baseline Configuration:** For clusters with up to 200 nodes and 4000 pods, each delegate should be configured with 2 vCPUs and 8 GB of memory.
```

> - **Incremental Scaling:** For every additional 50 nodes and 1000 pods, the delegate capacity should be increased by 0.5 vCPUs and 2 GB of memory

Reviewer: ```suggestion
- **Incremental Scaling:** For every additional 50 nodes and 1000 pods, the delegate capacity should be increased by 0.5 vCPUs and 2 GB of memory.
```

This scaling ensures that the delegate can handle the increased load and continue to collect metrics efficiently.

> Single Replica Requirement:

Reviewer: ```suggestion
**Single replica requirement:**
```

- All specified resource requirements pertain to a single replica of the delegate.

> - Instead of utilizing Horizontal Pod Autoscaler (HPA) to increase the number of smaller-sized replicas we recommend provisioning of each individual delegate with the necessary resources to handle the specified number of nodes and pods.

Reviewer: ```suggestion
- Instead of utilizing Horizontal Pod Autoscaler (HPA) to increase the number of smaller-sized replicas Harness recommends provisioning each delegate with the necessary resources to handle the specified number of nodes and pods.
```
